yt_dlp/extractor/getcourseru.py

```diff
@@ -8,7 +8,7 @@
 
 
 class GetCourseRuPlayerIE(InfoExtractor):
-    _VALID_URL = r'https?://player02\.getcourse\.ru/sign-player/?\?(?:[^#]+&)?json=[^#&]+'
+    _VALID_URL = r'https?://(player02\.getcourse\.ru|cf-api-2\.vhcdn\.com)/sign-player/?\?(?:[^#]+&)?json=[^#&]+'
```

Comment:
this needs to be non-capturing
```suggestion
    _VALID_URL = r'https?://(?:player02\.getcourse\.ru|cf-api-2\.vhcdn\.com)/sign-player/?\?(?:[^#]+&)?json=[^#&]+'
```